internal/checker/terraform/rules/azure/azurerm_app_service_min_tls_version.rego

```diff
@@ -0,0 +1,61 @@
+# METADATA
+# title: "Make sure that the web app is employing the most up-to-date version of TLS encryption"
+# description: "Confirm that your web application is utilizing the latest TLS encryption version to ensure enhanced security for data transmission"
+# related_resources:
+# - https://registry.terraform.io/providers/hashicorp/azurerm/latest/docs/resources/app_service
+# custom:
+#   id: CB_TFAZR_060
+#   severity: MEDIUM
+package lib.terraform.CB_TFAZR_060
+
+import future.keywords.in
+
+supportedResources := [
+	"azurerm_linux_web_app",
+	"azurerm_windows_web_app",
+	"azurerm_app_service",
+]
+
+isvalid(block) {
+	block.Type == "resource"
+	some label in block.Labels
+	label in supportedResources
+}
+
+resource[resource] {
+	some block in pass
+	resource := concat(".", block.Labels)
+}
+
+resource[resource] {
+	some block in fail
+	resource := concat(".", block.Labels)
+}
+
+pass[resource] {
+	some resource in input
+	isvalid(resource)
+	some innerBlock in resource.Blocks
+	innerBlock.Type == "site_config"
+	innerBlock.Attributes.min_tls_version >= 1.0
```

Comment:
covert to number for more accurate result
 to_number(innerBlock.Attributes.min_tls_version) >= 1.0

Reply:
Noted on this. Thank you. I'll push the changes